tests/floresta_cli_getmemoryinfo.py

```diff
@@ -0,0 +1,105 @@
+"""
+floresta_cli_getmemoryinfo.py
+
+This functional test cli utility to interact with a Floresta node with `getmemoryinfo`
+"""
+
+import os
+import re
+import tempfile
+from test_framework.test_framework import FlorestaTestFramework
+from test_framework.floresta_rpc import REGTEST_RPC_SERVER
+
+
+class GetMemoryInfoTest(FlorestaTestFramework):
+    """
+    Test `getmemoryinfo` rpc call, by creating a node and get
+    some memory stats with `stats` and `mallocinfo` modes
+    """
+
+    nodes = [-1]
+
+    # pylint: disable=duplicate-code
+    data_dir = os.path.normpath(
+        os.path.join(
+            tempfile.gettempdir(),
+            "floresta-func-tests",
+            "florestacli-getmemoryinfo-test",
+            "node-0",
+        )
+    )
+
+    def set_test_params(self):
+        """
+        Setup the two node florestad process with different data-dirs, electrum-addresses
+        and rpc-addresses in the same regtest network
+        """
+        GetMemoryInfoTest.nodes[0] = self.add_node_settings(
+            chain="regtest",
+            extra_args=[
+                f"--data-dir={GetMemoryInfoTest.data_dir}",
+            ],
+            rpcserver=REGTEST_RPC_SERVER,
+        )
+
+    def test_mode_stats_ibd(self, node):
+        """
+        Test `getmemoryinfo stats` when node is in IBD.
+        It should return a dictionary with key(str)/value(int).
+        """
+        result = node.get_memoryinfo("stats")
+        assert "locked" in result
```

Comment:
These assertions will break on non-gnu systems. Ideally we would either make sure it works everywhere, or not run them depending on the platform.

If you change the assert to >= it will work.